src/ArrayComparer/ArrayComparerInterface.php

```diff
@@ -13,7 +13,7 @@
  * @license BSD-3-Clause
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
-interface ArrayComparer {
+interface ArrayComparerInterface {
```

Comment:
Not possible - this is a BIG breaking change. And IMO it is bad practice, explanation at https://verraes.net/2013/09/sensible-interfaces/

Reply:
Sorry, I was just fixing issues that `phpcs` gave me, based on **your** `phpcs.xml` standard.   
Btw, 2013 is almost a decade ago :)   
https://www.php-fig.org/bylaws/psr-naming-conventions/

Reply:
The reasoning is what matters, not the age of the post :) Unfortunate that this got into PSR naming.